## GitHub Repo Update Submission 1

I am having some issues with GitHub. I have successfully downloaded my data into my FastX environment, but I need help to push my data to the repository properly. Currently, only a poorly formatted version of my Homework 1 is available in my repository. After trying to create a new directory (to copy over the README files and my FASTA downloads), I was denied all access to my repository, which I struggled with initially in class. 

I plan to attend office hours to overcome my roadblocks with GitHub. I plan to re-connect a new FastX directory I created to my GitHub with supervision in order to find any mistaken steps, which will be important to address when I continue to push more data in the future. Regarding my README formatting and GitHub organization, this will take time and practice to become comfortable with editing in the plaintext environment and creating well-organized folders within FastX to show up on GitHub. 

## GitHub Repo Update Submission 2

With the help of Professor He, I now feel comfortable pushing documents to my GitHub repo. The main issue was that I created a separate repository from class one, making syncing difficult. After finding this issue, I’ve been able to display my Homework markdown files and my raw sequence data. In terms of organization, I think my repository looks fine with its few entries, but I’m willing to move things around if needed. I also need to look into how to upload images to my homework files better. 

# Searching for the taxonomic classification of the protein JIP3: Organizing its Relatives. 

All information and work from this assignment will be applied to classifying the scaffolding protein family JIP and its close relatives and familial history. Protein sequence information will be taken from BLAST (BLASTp). The reference figure is taken from Dietmar Kültz, Evolution of Osmosensory MAP Kinase Signaling Pathways, American Zoologist. 

## Introduction and Background

JIP3 comes from the JIP family of MAPK scaffolding proteins. MAPK signaling affects axon susceptibility to pathological degeneration, yet the specific ways MAPK controls these outcomes are unknown. The JIP proteins have a scaffolding role in MAPK signaling. Little is known about the JIP scaffolding proteins, but JIP3, when inhibited, is found to slow down the degeneration of axons in response to injury or chemotherapy-induced neuropathy. Due to the need for more information on the function of the JIPs, constructing a taxonomic classification of the proteins would be a useful source of information. The JIP family contains other scaffolding proteins like JIP1, JIP2, and JIP4. Although JIP3 has been shown to limit degeneration more than its other family members, the Summers lab continues to measure neurodegeneration conditions for each JIP protein, including inhibiting groups of JIP proteins at one time. 

To document and visualize the varying relatedness of each JIP protein, I want to create a phylogenic tree to create a list of proteins related to JIP3, which could help develop a better understanding of its degenerative function. If I have sufficient time after I’ve created a satisfactory tree output, I’d like to repeat the same steps with the remaining JIPs 1,2 and 4. The Summers lab uses mouse embryos as a model for the human scaffolding proteins so I will conduct my BLAST search for related human proteins. 

The article Evolution of Osomosensory MAP Kinase Signaling Pathways by Dietmar Kultz describes a study in which the author’s goal was to collect more information on the cellular integration of information via MAP kinases to supplement ideas about cell dynamics/adaptation. My project follows a different route in terms of the target protein. Still, the steps taken toward the goal are similar, involving the collection of evolutionary history and relationships to supplement the study of these protein processes. My project will not involve multiple eukaryotes (only humans). Still, I will find the similarities between MAP kinase and JIP and the figure 2 image generation to be decent representatives of my project’s goal result. 

/https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/az/41/4/10.1093/icb/41.4.743/2/i0003-1569-041-04-0743-f02.jpeg?Expires=1683151817&Signature=FAc5Vs5L517KTIBiW6Nu-IfR9a9q1fOqvMSn7bJzxzRFvJfHw13dtdFdjy4cWQZ7y6c46lOujENbiI-IN0aQUp-S6iUanzaPEG3HyqndiTixbbVurcaTdJN3vyCJrrqD7lvycxAJjaIaNw6f8MLxrfiAWoDw0P4If2jpZN3FoPwyBrsuK4AvZPQTyqzui1prdmy0So9bkqnDpV6szunMaQaurNCsTj8P8P98I9pMR4AeipWwTjQUaJ6t7DEhYrMgfo3eV-7BhFtWLj-hW~5dUNEaTfkpu0mxN3asFylz2h48pUC5QV~uzFeq~lsgl7MVok7uESH4hBfviJg9KF1XFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=31744401&ar=2045585&xsltPath=~/UI/app/XSLT&imagename=&siteId=5294

**Fig 2. Radial phylogenetic tree of MAP kinases from C. Elegans and H. Sapiens.
Side B shows paralogous MAP kinases in Homo Sapiens, and side A shows the same in Caenorhabditis elegans. The human tree shows examples of subfamilies within JNK whose activation is facilitated by the JIP proteins. The alignment and analysis process to create this comparison was meant to provide input about the evolutionary history of MAP kinases.

## Reference

Dietmar Kültz, Evolution of Osmosensory MAP Kinase Signaling Pathways, American Zoologist, Volume 41, Issue 4, August 2001, Pages 743–757, https://doi.org/10.1093/icb/41.4.743

## Expected Figure

The expected figure will look similar to fig 2 but in a linear, cladogram form. Suppose my initial run with my selected query sequence is successful. In that case, I plan to produce more of a similar figure for other proteins studied in the lab or expand my first figure to include more distant relatives. 

## Materials and Methods 

The first step is to run a BLAST search for JIP3 with human limitations. Based on the identity score and relevance of the output to the research laboratory, I will select the sequences which show the best percent identity and E-value. Next, I will manually select those sequences and download them as FATSA files to my FastX environment (and GitHub). Based on those sequences, I will need to develop a way for FastX to compare and align the sequences to determine their placement on the phylogenetic tree. These alignment and tree generation steps need more work to show JIP3 and the proteins to which it has the strongest relation. 

More research needs to be done on my end as to whether this process will work in the FastX environment alone or if I need to implement new tools like MEGA. I will use the book Phylogenetic Trees Made Easy by Barry G. Hall to pinpoint a set of materials and methods better. 

